cli/Cargo.toml

```diff
@@ -64,6 +64,7 @@ hex = { workspace = true }
 indexmap = { workspace = true }
 itertools = { workspace = true }
 jj-lib = { workspace = true }
+libssh2-sys = { workspace = true }
```

Comment:
This dependency could be made Windows-only by moving it to a `[target.'cfg(windows)'.dependencies]` section.

Reply:
Will this work for the openssl-on-win32 feature? The feature needs to be defined for all targets even if it's a no-op everywhere but Windows